libraries/adaptive-expressions/package.json

```diff
@@ -50,13 +51,15 @@
     "build": "tsc",
     "build-docs": "typedoc --theme markdown --entryPoint adaptive-expressions --excludePrivate --includeDeclarations --ignoreCompilerErrors --module amd --out ..\\..\\doc\\adaptive-expressions .\\lib\\index.d.ts --hideGenerator --name \"Bot Builder SDK - Expression\" --readme none",
     "clean": "erase /q /s .\\lib",
+    "postbuild": "browserify lib/index.js -s AEL -o dist/adaptive-expressions.js",
```

Comment:
Can we just place this inside `lib/browser.js`? Also, is it worth adding `"browser": "lib/browser.js"` to `package.json`?

Reply:
Placed it in lib/browser.js file. 
I have tried adding browser field in package json, but it throws error that it cant walk the dependecy graph. Not sure why.

Reply:
I ran into that as well, I can resolve once this gets merged.

Reply:
Thanks